src/store/edge-device.js

```diff
@@ -0,0 +1,29 @@
+import { FETCH_EDGE_DEVICE_DETAILS } from './action-types.js'
+import { EDGE_DEVICE_DETAILS } from './mutation-types'
+
+const state = {
+  version: require('@/../package.json').version
```

Comment:
@vickywane version here should not default to the PWA version. It should be undefined until the PWA connects to an edgeDevice and fetches its version.

Reply:
I have made this correction. 

It now defaults to `null` until it's loaded. However, would it not be better to have a default loading text for UX?